front/form.form.php

```diff
@@ -142,21 +142,13 @@
 
       // redirect to created item
       if ($_SESSION['glpibackcreated']) {
-         $item_ticket = new Item_Ticket;
-         $tickets = $item_ticket->find([
-            'itemtype' => PluginFormcreatorFormAnswer::class,
-            'items_id' => $formAnswer->getID(),
-         ]);
-
-         if (count($tickets) === 1) {
-            $current_ticket = array_pop($tickets);
-            $tickets_id = $current_ticket['tickets_id'];
-            Html::redirect(Ticket::getFormURLWithID($tickets_id));
-         }
-
-         if (count($tickets) > 1) {
-            Html::redirect(PluginFormcreatorFormAnswer::getFormURLWithID($formAnswer->getID()));
+         print_r($formAnswer->targetList);
```

Comment:
I guess this is for debug purpose and should be removed

Reply:
oh yes :p

Reply:
Fixed ;)